dataflow/src/main/java/org/checkerframework/dataflow/analysis/BackwardAnalysisImpl.java

```diff
@@ -206,6 +206,7 @@ protected void initInitialInputs() {
     List<ReturnNode> returnNodes = cfg.getReturnNodes();
     assert transferFunction != null : "@AssumeAssertion(nullness): invariant";
     S normalInitialStore = transferFunction.initialNormalExitStore(underlyingAST, returnNodes);
+    assert transferFunction != null : "@AssumeAssertion(nullness): invariant";
```

Comment:
This is a duplicate of the assert on line 207.

Reply:
Addressed by adding `@SideEffectFree`.